tests/integration/helpers/a11y-audit-if-test.js

```diff
@@ -16,24 +18,23 @@ module('Integration | Helper | a11yAuditIf', function (hooks) {
     assert.ok(true, 'a11yAuditIf should not run a11yAudit');
   });
 
-  test('a11yAudit should execute a11yAudit if enableA11yAudit=ture is passed as query param', async function (assert) {
-    await render(hbs`{{#axe-component}}<button></button>{{/axe-component}}`);
+  test('a11yAudit should execute a11yAudit if enableA11yAudit=true is passed as query param', async function (assert) {
+    try {
+      await render(hbs`{{#axe-component}}<button></button>{{/axe-component}}`);
 
-    // TODO
-    utils.getLocation = function () {
-      return {
-        search: '?enableA11yAudit=true',
-      };
-    };
+      setEnableA11yAudit(true);
 
-    try {
-      await a11yAuditIf(this.element);
-      assert.ok(false, 'audit should have failed');
-    } catch (error) {
-      let foundExpectedError = error.message.startsWith(
-        'The page should have no accessibility violations.'
-      );
-      assert.ok(foundExpectedError);
+      try {
+        await a11yAuditIf(this.element);
+        assert.ok(false, 'audit should have failed');
+      } catch (error) {
+        let foundExpectedError = error.message.startsWith(
+          'The page should have no accessibility violations.'
+        );
+        assert.ok(foundExpectedError);
+      }
```

Comment:
Can also be moved to `assert.rejects` (seems totally fine for a future PR)

Reply:
Fixed it for completeness.